### DISCUSSION TITLE
How to let me automatically close the navbar when I click the navbar to switch the page to the next page?

### DISCUSSION THREAD

**Discussion by sxadxsx** [themesberg/flowbite-react#1131] [Q&A]
How to let me automatically close the navbar when I click the navbar to switch the page to the next page?

**Comment by phoskee** [comment]
```py
const location = useLocation();

  useEffect(() => {
    const navbarCollapse = document.getElementById("AAA");
    if (navbarCollapse) {
      navbarCollapse.classList.add("hidden");
    }
    console.log("URL changed:", location.pathname);
  }, [location.pathname]);

...

<NavbarCollapse id="AAA">
```
Using this "trick" helps achieve the goal, but it makes the minus button unstable (you have to click it twice to open)
I also tried to bind the boolean of the hidden props,` <NavbarCollapse hidden={{someVAR}>` but if it is changed there it doesn't open anymore.
